packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergJsException.kt

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.mobile.ReactNativeGutenbergBridge
+
+import com.facebook.react.bridge.ReadableMap
+
+data class GutenbergJsException(
+    var type: String,
+    var value: String,
+    var stackTrace: List<GutenbergJsExceptionStackTraceElement>,
+    var context: Map<String, Any> = emptyMap(),
+    var tags: Map<String,String> = emptyMap(),
+    var isHandled: Boolean,
+    var handledBy: String
+){
+    companion object {
+        @JvmStatic
+        fun fromReadableMap(rawException: ReadableMap): GutenbergJsException {
+            val type: String = rawException.getString("type") ?: ""
+            val value: String = rawException.getString("value") ?: ""
+            val stackTrace: List<GutenbergJsExceptionStackTraceElement> = rawException.getArray("stackTrace")?.toArrayList()?.map {
+                val stackTraceElement = it as ReadableMap
+                GutenbergJsExceptionStackTraceElement(
+                    stackTraceElement.getString("fileName") ?: "",
+                    stackTraceElement.getInt("lineNumber"),
```

Comment:
The name of this property in JavaScript is `lineno`. Sharing in case this produces an issue when parsing the object on Android.
https://github.com/WordPress/gutenberg/blob/b61200e6400aa5ad12cd3c84a3d33b9d63ac41a7/packages/react-native-bridge/lib/parseException.js#L89